components/HomepageSlider/HomepageSlider.module.scss

```diff
@@ -0,0 +1,53 @@
+.slider {
+    top: 50%;
+    left: 50%;
+    transform: translate(0);
+    width: 588px;
+    height: 248px;
+    .slides {
+        transition: left .75s;
+        display: flex;
+        position: relative;
+        top: 0;
+        left: -588px;
+        width: 10000px;
+        .slide {
+            width: 588px;
+            height: 248px;
+            cursor: pointer;
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: center;
+            transition: all 1s;
+            position: relative;
+            blockquote,
+            hr,
+            p {
+                pointer-events: none;
+            }
+        }
+    }
+}
+
+[data-state~="responsive.mobileS"],
+[data-state~="responsive.mobileM"] {
+    .slider {
+        height: 190px;
+    }
+}
+
+[data-state*="responsive.mobile"] {
+    .slider {
+        width: 248px;
```

Comment:
Does this have to be a static value? can we make it dynamic to make sure it is responsive enough for both desktop and mobile?

Reply:
Currently there are 2 values that changes the size of the slider, 248px for mobile and 588px on desktop screens. Do you mean if the slide size should be in percentages so that is steadily grows and shrinks?

Reply:
Yes, or having min or max values

Reply:
I changed it now using view width (vw). Seems to work perfectly this way.